ui/raidboss/data/05-shb/raid/e1s.js

```diff
@@ -600,16 +600,53 @@
       'locale': 'cn',
       'replaceSync': {
         'Engage!': '战斗开始！',
-        'Eden Prime': 'Eden Prime',
-        'Arcane Sphere': '立体魔法阵',
+        'Eden Prime': '',
+        'Arcane Sphere': '',
       },
       'replaceText': {
         'attack': '攻击',
-        'Unknown Ability': 'Unknown Ability',
+        'Vice of Vanity': '',
+        'Vice of Thievery': '',
+        'Vice of Sloth': '',
+        'Vice of Pride': '',
```

Comment:
Did you mean to leave all of these blank?

Reply:
I was trying to fill these but it seems these translations are not released in current exd/csv files of CN client, so I just leave them blank and will finish them when savage is released.

Reply:
Ok! I just figured they'd all mostly be the same as e1n, but waiting is fine!

Reply:
Yeah mostly, but not all of them, I need to update them eventually so I’m going to wait till then.